crates/dm-langserver/src/debugger/mod.rs

```diff
@@ -1089,7 +1093,7 @@ handle_request! {
             DebugClient::Extools(extools) => {
                 let extools = extools.get()?;
 
-                if params.variablesReference >= 0x01_000000 {
+                if params.variablesReference >= 0x0100_0000 {
```

Comment:
I'm not particularly fussed here but the strange _ from before would likely have been separating the byond type id from the object identifier.

1 byte for the type, 3 bytes for the rest

Reply:
i was wondering if that was the case but the fact that there's no comments anywhere in sight indicating that was a pretty big 🤷🏻